roles/cpu/tasks/cpu.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Execute CPU healthcheck command on Cisco IOS XR device
+  cisco.iosxr.iosxr_command:
+    commands:
+      - "{{ cpu_healthcheck_command }}"
+  register: cpu_check
+
+- name: Display CPU check output
```

Comment:
Once we have all the healtcheck data related to CPU , we  need to parse this data as well and run the assertions based on use inputs (for example CPU threshold ).

This means we need to define parsers as well and we could use native parsers for this , you can take the reference from https://github.com/redhat-cop/network.bgp/blob/main/roles/health_checks/tasks/templates/eos_show_ip_bgp_summary.yaml  where we have used content template parsers , but for this we will use native parsers , for more details please check out : https://docs.ansible.com/ansible/latest/collections/ansible/utils/cli_parse_module.html